config/babel.prod.js

```diff
@@ -10,20 +10,20 @@
 module.exports = {
   babelrc: false,
   presets: [
-    'babel-preset-es2015',
-    'babel-preset-es2016',
-    'babel-preset-react'
-  ].map(require.resolve),
+    require.resolve('babel-preset-es2015'),
+    require.resolve('babel-preset-es2016'),
+    require.resolve('babel-preset-react')
+  ],
   plugins: [
-    'babel-plugin-syntax-trailing-function-commas',
-    'babel-plugin-transform-class-properties',
-    'babel-plugin-transform-object-rest-spread',
-    'babel-plugin-transform-react-constant-elements',
-  ].map(require.resolve).concat([
+    require.resolve('babel-plugin-syntax-trailing-function-commas'),
+    require.resolve('babel-plugin-transform-async-to-generator'),
```

Comment:
Out of curiosity, is there a reason we use `transform-async-to-generator` to first transform async functions to generators and then regenerator to transform them to ES5? (Regenerator can also transform async functions directly, in which case the async-to-generator won't be needed.)

I've tested the Regenerator-only solution with following config:
```js
    require.resolve('babel-plugin-syntax-async-functions'),
    [require.resolve('babel-plugin-transform-regenerator'), {
      asyncGenerators: false,
      generators: true,
      async: true
    }],
```
and it compiled to more or less the same code except it uses only the regenerator runtime and not the `_asyncToGenerator` helper from Babel. I don't know, if there are some minor differences in the runtime behaviour of these alternatives.

If you want to switch, I can send a pull request with this config I have.

P.S. It's very cool to have async/await supported by create-react-app! This was the reason I had to eject my config and I've been planning to make a PR to add them, but you beat me to it 😎